scopes/harmony/node/node.env.ts

```diff
@@ -21,4 +24,8 @@ export class NodeEnv implements DependenciesEnv {
       splitComponentBundle: false,
     };
   }
+
+  getPackageJsonProps() {
+    return this.tsAspect.getCjsPackageJsonProps();
```

Comment:
in aspect env you accomplish this with
```
this.reactEnv.getCjsPackageJsonProps();
```

let's use the same code, if possible

Reply:
done